### ISSUE TITLE
Improve UI/copy on the MV3 user gesture dialog

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#8301]
Context
---
- Current copy: "Please click "OK" to allow PixieBrix to open the sidebar."
 
Definition of Done
---
- Change copy to 'Please click "Open the Sidebar" to open the mod sidebar'
- Change button text to "Open Sidebar"
- "PixieBrix" branding removed from the dialog

**Comment by BrandonPxBx** [comment]
@twschiller I'm focused on Deployments, Mod Toothbrush users, and our internal research panel recruitment operations. Is this a higher priority than those?

**Comment by BrandonPxBx** [comment]
I ran two preference tests. The first was on the current copy and the copy proposed in the Definition of Done. 

There was no statistically significant difference between the two. The definition of done copy received 52% of the vote from 25 test participants.

**Comment by BrandonPxBx** [comment]
[Here's the study](https://app.lyssna.com/tests/2a9caccc9faa/results/e83eb01eb57e)

**Comment by BrandonPxBx** [comment]
@twschiller I thought the discussion we had in our 1 on 1 yesterday was that we just wanted to remove the word "PixieBrix" and that the UI was fine for now?

**Comment by BrandonPxBx** [comment]
> > @twschiller I thought the discussion we had in our 1 on 1 yesterday was that we just wanted to remove the word "PixieBrix" and that the UI was fine for now?
> 
> Correct - this issue is in the "Icebox". We're just tracking it as something to consider prior to MV3 switchover. If we decide not to, we'll close this issue. There's no action to take on this issue right now

Got it, that's not displayed anywhere on this UI from what I can tell and I just saw this from my notifications pane.

**Comment by grahamlangford** [comment]
@twschiller I don't think we ever settled on Acceptance Criteria for this. Do you have any more specific guidance? The styles are completely isolated and outside react, so we can't just use the react-bootstrap modal and button components
